lib/bots/versionbot.ts

```diff
@@ -650,17 +650,21 @@ export class VersionBot extends ProcBot {
                 return false;
             });
         }).then((exists: boolean) => {
-            let versionistCommand = 'versionist';
-            if (exists) {
-                versionistCommand = `${versionistCommand} -c versionist.conf.js`;
-                this.logger.log(LogLevel.INFO, 'Found an overriding versionist config ' +
-                    `for ${versionData.repoFullName}, using that`);
-            }
-
-            return Promise.mapSeries([
-                versionistCommand,
-                'git status -s'
-            ], cliCommand);
+            let versionistCommand: string;
+            return this.getNodeBinPath().then((nodePath: string) => {
+                versionistCommand = `${nodePath}${path.sep}versionist`;
```

Comment:
Nit: imo this is easier & clearer as just `path.join(nodePath, 'versionist')`. Also handles some cases like normalizing out any extra slashes (e.g. on the end of `nodePath`). Don't see how those could come up, but maybe that's useful :man_shrugging: